src/Wallabag/CoreBundle/Controller/EntryController.php

```diff
@@ -225,21 +225,24 @@ private function showEntries($type, Request $request, $page)
     {
         $repository = $this->get('wallabag_core.entry_repository');
 
+        $sortBy = $request->get('sort', 'id');
+        $direction = $request->get('direction', 'DESC');
```

Comment:
You should filter values of both direction & sort to avoid sql injection in the `getBuilderFor` method